### ISSUE TITLE
Where can I find a tutorial on janusgraph cluster deployment

### ISSUE THREAD

**Issue by czs1993** [JanusGraph/janusgraph#1641]
I didn't find a deployment tutorial for janusgraph from the official documentation,Where can I find a tutorial on janusgraph cluster deployment or Single deployment?

**Comment by porunov** [comment]
@czs1993 Issues are used only for bug reports and feature requests. You should ask such questions in this group: https://groups.google.com/forum/#!forum/janusgraph-users

You can find getting started tutorial here: https://docs.janusgraph.org/latest/getting-started.html
Also, there are nice tutorials here: https://developer.ibm.com/dwblog/2018/whats-janus-graph-learning-deployment/

The more specific your question the more help can community provide to you. For example, if you are looking to deploy JanusGraph with specific storages or index storages then you should mention it also. If you need to make specific scheme and optimize some traversal by including indexes then you should mention it also.

In short, JanusGraph cluster deployment is the same as JanusGraph single server deployment. JanusGraph instances don't talk to each other directly but they can share some metadata through the storage (like IDs block allocation, index creation, re-indexing and so on).
